src/slider/slider.scss

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-baseline; // 11px
+
+@mixin iui-slider-component-container {
+  display: flex;
+
+  .iui-slider-min,
+  .iui-slider-max {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: $iui-xs;
+    }
+  }
+
+  .iui-slider-min {
+    margin-right: $iui-sm;
+    text-align: right;
+  }
+
+  .iui-slider-max {
+    margin-left: $iui-sm;
+  }
+
+  &.iui-disabled {
+    @include iui-slider-disabled;
+  }
+}
+
+@mixin iui-slider-container {
+  position: relative;
+  flex-grow: 1;
+  pointer-events: all;
+  cursor: pointer;
+
+  .iui-slider-rail {
+    position: absolute;
+    width: 100%;
+    height: $track-height;
+    top: $iui-sm;
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+  }
+
+  .iui-slider-thumb {
+    position: absolute;
+    pointer-events: all;
+    height: $thumb-height;
+    width: $thumb-height;
+    border-radius: 100%;
+    top: $track-height + 1; // position center of thumb in center of track
+    transform: translateX((-$thumb-height - 2) * 0.5); // Adjust the left (set in code) by radius of thumb so thumb center is at value position
+    z-index: 1;
+    cursor: grab;
+    @include themed {
+      background: t(iui-color-background-1);
+      border: 1px solid t(iui-color-background-5);
+    }
+
+    &:active {
+      cursor: grabbing;
+    }
+
+    @include iui-focus();
+  }
+
+  .iui-slider-track {
+    pointer-events: none;
+    position: absolute;
+    height: $track-height;
+    left: 0;
+    top: $iui-sm;
+    @include themed {
+      background-color: t(iui-color-foreground-primary);
+    }
+  }
+}
+
+@mixin iui-slider-ticks {
+  position: relative;
+  padding-top: $iui-m;
+  display: flex;
+  pointer-events: none;
+  justify-content: space-between;
+  user-select: none;
+
+  .iui-slider-tick {
+    position: relative;
+    display: flex;
+    pointer-events: none;
+    justify-content: center;
+    width: 1px;
+    height: $tick-height;
+    line-height: $tick-height * 4.25;
```

Comment:
is this what the value is supposed to be? 4.25 seems a bit random

Reply:
@bsteinbk you know this one?

Reply:
It was determined by trial and error.... it is defined to provides enough room for the ticks to be drawn above the text. Maybe it should be based on em and not px - a value of 3em also looks good.

Reply:
we plan to switch to rem units in the future, but we cant have a mix of them.

My concern here is that a line height of 46.75px could cause problems on different screen sizes.

Reply:
Changed to 44px...